packages/did-core-test-server/src/services/scenarios/equivalence-properties/equivalence-properties-tests.js

```diff
@@ -0,0 +1,65 @@
+
+const {
+  generateScenarioResults
+} = require('../../utils');
+
+const getMethod = id => id.split(':')[1];
+
+// WIP for other assertions
```

Comment:
These commented blocks are ok IMO, will expect them to get uncommented / removed when the other assertions are tested.